sys/include/ztimer/periodic.h

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2020 Kaspar Schleiser <kaspar@schleiser.de>
+ *               2020 Freie Universität Berlin
+ *               2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+/**
+ * @ingroup     sys_ztimer
+ * @brief       Periodic ztimer API
+ *
+ *
+ * Once started, the periodic timer will call the configured callback function
+ * once each interval until the timer is either stopped using ztimer_periodic_stop
+ * or the callback function returns a non-zero value.
+ *
+ * Example:
+ *
+ * ```
+ * #include "ztimer/periodic.h"
+ *
+ * static void callback(void *arg)
+ * {
+ *    puts(arg);
+ * }
+ *
+ *
+ * int main()
```

Comment:
```suggestion
 * int main(void)
```

I'm aware that no arguments only results in a function template at the declaration of a function, not at the implementation. But still, I'm personally in favor of consistently adding `void` as parameter in order to not having to remember the more obscure parts of the C standard.

Reply:
sure. I didn't compile the example code, there might be other issues... will fix